<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brightness</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc/rsc.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>197</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>300000</number>
      </property>
      <property name="maximum">
       <number>2948310</number>
      </property>
      <property name="sliderPosition">
       <number>1500000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSafe"/>
    <addaction name="actionTo_Tray"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSafe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Safe</string>
   </property>
  </action>
  <action name="actionTo_Tray">
   <property name="text">
    <string>To Tray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>horizontalSlider</tabstop>
 </tabstops>
 <resources>
  <include location="../rsc/rsc.qrc"/>
 </resources>
 <connections/>
</ui>
